packages/core/contracts/data-verification-mechanism/implementation/FixedSlashSlashingLibrary.sol

```diff
@@ -9,58 +9,64 @@ import "../interfaces/SlashingLibraryInterface.sol";
  elaborate slashing algorithms via UMA governance.
  */
 
-contract SlashingLibrary is SlashingLibraryInterface {
+contract FixedSlashSlashingLibrary is SlashingLibraryInterface {
```

Comment:
`SlashingLibrary` was also referenced in `packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts` that should be updated there as well